subsys/bluetooth/mesh/blob_cli.c

```diff
@@ -1651,3 +1679,8 @@ bool bt_mesh_blob_cli_is_busy(struct bt_mesh_blob_cli *cli)
 {
 	return cli->state != BT_MESH_BLOB_CLI_STATE_NONE;
 }
+
+void bt_mesh_blob_cli_set_chunk_interval_ms(struct bt_mesh_blob_cli *cli, uint32_t interval_ms)
+{
+	cli->chunk_interval_ms = interval_ms;
```

Comment:
Not sure if you have discussed this, but I think we could introduce a Kconfig option to set up the delay at compile time. This way some implementations may get rid of this call at a start.

Reply:
I wasn't there when the task was groomed, so I am not sure. But if I understood it correctly this is most useful during runtime, as then the user can adjust this delay/interval depending on what else happens on the network (probably using some dynamic algorithm fitted to their network), plus if nodes get added to the network (changing the "ideal average interval/delay" for a lack of better words) recompilation isn't needed. 
The idea for future used, described in the task, is to add a nordic vendor model to access this function (as goal is modifiability during runtime).

Reply:
I'm not against the runtime configuration. But feel reasonable to add a Kconfig option as well. It can even be by default set to some non-zero value so when customers test DFU, it doesn't disrupt the network and doesn't gives bad impression on the feature. But ok, if you feel it is not needed, we can add this later if we get such request from customers.

Reply:
I am not necessarily against it, just if I add it I'd like to keep default behaviour (so delay == 0), and then I don't see much value. If we are both fine with/without kconfig for now, maybe @alxelax or @HaavardRei have opinions on whether to add this as kconfig or not?

Reply:
I do not have a strict opinion about this. Up to you guys.

Reply:
> I'd like to keep default behaviour (so delay == 0), and then I don't see much value

It can be adjusted in our samples at least.

Reply:
Sounds like it's nice to have, so if it isn't too much hassle you can add it. Else create a task for it then we'll prioritize it later :)